scoring/scoring.py

```diff
@@ -34,6 +34,7 @@
 import matplotlib.pyplot as plt
 import numpy as np
 import pandas as pd
+from tabulate import tabulate
```

Comment:
Where is this used?

Reply:
Ah good catch, thanks, I used that for debugging I think. Removing.